plugins/modules/mysql_role.py

```diff
@@ -121,6 +121,15 @@
     type: bool
     default: true
 
+  column_case_sensitive:
+    description:
+      - When C(true), the module won't uppercase the field in the privileges.
+      - When C(false), the field names will be upper cased. This was the default before this
+        feature was introduced but since MySQL/MariaDB is case sensitive you should set this
+        to C(true) in most cases.
```

Comment:
```suggestion
    description:
      - The default is C(false).
      - When C(true), the module will not uppercase the field in the privileges.
      - When C(false), the field names will be upper-cased. This was the default before this
        feature was introduced but since MySQL/MariaDB is case sensitive you should set this
        to C(true) in most cases.
```